scvi/module/_jaxpeakvae.py

```diff
@@ -0,0 +1,201 @@
+from typing import Dict
+
+import jax
+import jax.numpy as jnp
+import numpyro.distributions as dist
+from flax import linen as nn
+
+from scvi import REGISTRY_KEYS
+from scvi.module.base import JaxBaseModuleClass, LossRecorder
+
+from ._jaxvae import Dense
+
+
+class FlaxEncoder(nn.Module):
+    n_input: int
+    n_latent: int
+    n_hidden: int
+    dropout_rate: int
+
+    def setup(self):
+        self.dense1 = Dense(self.n_hidden)
+        self.dense2 = Dense(self.n_hidden)
+        self.dense3 = Dense(self.n_latent)
```

Comment:
provide more descriptive names given that these are not sequential like the names would suggest otherwise